Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -14,25 +14,40 @@ Check for expected values and outcomes in tests.
 
 ## Overview
 
-Tests may validate behaviors using expectations. This page describes the various
-built-in expectation APIs.
+The testing library provides `#expect()` and `#require()` macros you use to 
+validate expected outcomes and a `Confirmation` type you can use to confirm the 
+occurrence of asynchronous events. To validate that an error is thrown, or _not_ thrown, 
+the testing library provides several overloads of the macros that you can use.
 
 ## Topics
 
-### Validating behavior using expectations
+### Evaluating conditions
 
 - ``expect(_:_:sourceLocation:)``
+
+### Requiring conditions
+
 - ``require(_:_:sourceLocation:)-5l63q``
 - ``require(_:_:sourceLocation:)-6w9oo``
 
-### Validating asynchronous behavior using confirmations
+### Confirming asynchronous events
 
-- ``Confirmation``
 - ``confirmation(_:expectedCount:fileID:filePath:line:column:_:)``
+- ``Confirmation``
+
+### Evaluating errors
```

Comment:
We should consider keeping pairs of `#expect()` and `#require()` grouped together under the same subheading. `#require()` behaves exactly the same as `#expect()` except that it throws on failure.